cobigen-eclipse/cobigen-eclipse-test/src/main/java/com/devonfw/cobigen/eclipse/test/OpenAPITest.java

```diff
@@ -61,76 +69,85 @@ public static void setupClass() throws Exception {
   @Test
   public void testBasicOpenAPIGeneration() throws Exception {
 
-    // copy sample project to external location and import it into the workspace
-    String testProjName = "ExtTestProj";
-    IJavaProject project = this.tmpMavenProjectRule.createProject(testProjName);
-    FileUtils.copyFile(new File(resourcesRootPath + "input/devonfw.yml"),
-        project.getUnderlyingResource().getLocation().append("devonfw.yml").toFile());
-    project.getProject().refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
-    this.tmpMavenProjectRule.updateProject();
+    this.currentHome = this.tmpFolderRule.newFolder("cobigen-test-home").toPath();
+    withEnvironmentVariable(ConfigurationConstants.CONFIG_ENV_HOME, this.currentHome.toString()).execute(() -> {
+      // disable Generated annotation
+      Path propertiesPath = this.currentHome.resolve(ConfigurationConstants.COBIGEN_CONFIG_FILE);
+      try (PrintStream out = new PrintStream(new FileOutputStream(propertiesPath.toString()))) {
+        out.print(ConfigurationConstants.ADD_GENERATED_ANNOTATION + "=false");
+      }
```

Comment:
Why is this test being modified?
Wouldn't it be better to write a new test just for the generated annotation instead?